plugins/amazonq/chat/jetbrains-community/src/software/aws/toolkits/jetbrains/services/amazonq/toolwindow/AmazonQToolWindowFactory.kt

```diff
@@ -98,13 +97,12 @@ class AmazonQToolWindowFactory : ToolWindowFactory, DumbAware {
                 // note we name myProject intentionally ow it will shadow the "project" provided by the IDE
                 override fun onProfileSelected(myProject: Project, profile: QRegionProfile?) {
                     if (project.isDisposed) return
-                    AmazonQToolWindow.getInstance(project).disposeAndRecreate()
```

Comment:
TODO: send active profile to LSP so that LSP can render selected profile in a card